src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -339,6 +340,19 @@ private async Task<DurableOrchestrationStatus> GetDurableOrchestrationStatusAsyn
             return this.ConvertFrom(orchestrationState, historyArray);
         }
 
+        private OrchestrationStatus ConvertFrom(OrchestrationRuntimeStatus runtimeStatus)
+        {
+            OrchestrationStatus status;
+            if (Enum.TryParse<OrchestrationStatus>(runtimeStatus.ToString(), out status))
+            {
+                return status;
+            }
+            else
+            {
+                throw new ArgumentException($"The OrchestrationRuntimeStatus {runtimeStatus} is exptected to be converted into OrchestrationStatus");
```

Comment:
You don't need to convert back and forth from strings. Just do a direct cast like we do in other places:
```csharp
return (OrchestrationStatus)runtimeStatus;
```

Reply:
Oh. God... I've never known that. Thanks

Reply:
Done